BankUsers/FinancialAnalyst.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BankUsers
+{
+    public class FinancialAnalyst : OfficeWorker
```

Comment:
In my opinion, in this task you need to create enum with different jobs and then create  "OfficeWorkerType" property in OfficeWorker class